storage/scylla_api.c

```diff
@@ -443,7 +444,7 @@ static status_t create_identity_table(CassSession* session, bool need_drop) {
   }
   if (execute_query(session,
                     "CREATE TABLE IF NOT EXISTS identity("
-                    "id bigint, hash blob, status tinyint, PRIMARY KEY (id));") != CASS_OK) {
+                    "id bigint, time timestamp, hash blob, status tinyint, PRIMARY KEY (id));") != CASS_OK) {
```

Comment:
How about changing `time timestamp` into `ts`?

Reply:
OK! ts is better.